decidim/decidim
# Fix initiatives graphql type

#### :tophat: What? Why?
Fixing the initiative api type as at the moment it throws error, due to existance of Author field and some other Initiative missing / renamed fields. 
the bug has been introduced in  #6143 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Relates  #6143
- Fixes  #7036

#### Testing
Steps to reproduce the behavior:

- Go to Api, and add a quey like https://meta.decidim.org/api/graphiql?query=%7B%0A%20%20initiatives%20%7B%0A%20%20author%20%7B%0A%20%20%20%20id%0A%20%20%7D%0A%20%20%20%20avatarUrl%0A%20%20%20%20badge%0A%20%20%20%20deleted%0A%20%20%20%20initiativeSupportsCount%0A%20%20%20%20initiativeVotesCount%0A%20%20%20%20profilePath%0A%20%20%20%20offlineVotes%0A%20%20%7D%0A%7D%0A
- Observe error SyntaxError: JSON.parse: unexpected character at line 2 column 1 of the JSON data
- Apply the patch 
- Run the api query, remove the user related fields 
- Observe the working Initiative

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](../CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)

:hearts: Thank you!
